merge-headers/index.js

```diff
@@ -32,6 +36,10 @@ module.exports = function mergeHeaders(submitted, defaults) {
  * no return value
  */
 function iterate(iterator, fn) {
+    if (!(iterator instanceof Headers)) {
```

Comment:
What is the benefit in using `Headers`? 
maybe we should only use object since it also run in node?

Reply:
If someone polyfills Headers badly, it's really their problem

Reply:
fetch is a browser API. If it supports Headers, not supporting Headers is unexpected behaviour. If consumers use fetch in node, it should match the fetch API as well

Reply:
See the spec -> https://fetch.spec.whatwg.org/#headers-class

![image](https://user-im
ages.githubusercontent.com/8016250/38174839-5336a818-35dc-11e8-9feb-9d6d545718a5.png)

So as written it's perfectly valid to use plain object.

Reply:
You are confusing the definition of the headers option, and the Headers class - which dictates the implmentation of append, delete, has, and get.
If Headers class is supported, user is able, like in HTTP protocol, to apply multiple entries with the same key - which will not be possible using an object literal.